### DISCUSSION TITLE
How strict is the scikit_learn version requirement?

### DISCUSSION THREAD

**Discussion by fxcoudert** [mir-group/nequip#269] [General]
`setup.py` says: `"scikit_learn<=1.0.1",  # for GaussianProcess for per-species statistics; 1.0.2 has a bug!`

Do you have a link to what the bug is, and how strict that version requirement is? I'd like to see if I can use the 1.1.x versions, because they are preinstalled on our HPC center, and due to strong restrictions it is painful to install an older version.

**Comment by Linux-cpp-lisp** [comment]
@nw13slx ?

**Reply by fxcoudert** [reply]
May I ask again what the bug is, so we can check if it is fixed? We would really like to use the preinstalled version, it will make our life much simpler.

**Reply by nw13slx** [reply]
sorry that I missed this. I will check on my pull request that remove this dependency. #248

**Reply by nw13slx** [reply]
The pull request is merged to `develop` branch. Would you be able to use that?

**Reply by fxcoudert** [reply]
Thanks, we will look into that option.

**Reply by Linux-cpp-lisp** [reply]
FYI @fxcoudert it's not merged right now (https://github.com/mir-group/nequip/pull/277), but we hope to have it merged in the next day or two and will let you know as soon as we do.  This entirely removes the `sklearn` dependency thanks to @nw13slx 's efforts, so it should work for you.

**Reply by Linux-cpp-lisp** [reply]
OK @fxcoudert this is up on `develop` now, hopefully it entirely circumvents this issue!
